pkg/actions/testdata/lua/catalogexport_hive_partition_pager.lua

```diff
@@ -0,0 +1,69 @@
+local hive = require("lakefs/catalogexport/hive")
+
+-- helper function to slice table array
+function table.slice(tbl, first, last, step)
```

Comment:
AFAICT this implementation doesn't really slice a table array.  For instance if the table is sparse, it is still an array -- but its `table_slice` will be a dense array "table".

Reply:
changed name